independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/JandexUtil.java

```diff
@@ -415,4 +417,42 @@ public static boolean isImplementorOf(IndexView index, ClassInfo info, DotName n
         return isImplementorOf(index, superClass, name, additionalIgnoredSuperClasses);
     }
 
+    public static boolean isAssignableFrom(DotName name1, DotName name2, IndexView index) {
```

Comment:
Can we have better names than `name1` and `name2`? It's already hard enough to remember which of the two parameters should be subtype/supertype, this will be confusing later.

Reply:
Sure yeah

Reply:
Fixed